language-design.md

```diff
@@ -70,7 +158,17 @@ what ways do the themes apply to the study and creation of DSLs?
 
 **Response**
 
+The themes would not apply to DSLs as much. Because DSLs are domain
+specific, it’s likely that most DSLs are the only language to serving the domain. 
+Thus, there is already less or no competition between languages for DSLs. 
```

Comment:
I would call jQuery a DSL, and tons of people have a bias against it (mostly people who think others who use jQuery don't understand JavaScript and thus are bad programmers).